spec.bs

```diff
@@ -2737,11 +2782,15 @@ algorithms to achieve the outcomes described in the above explanatory content.
      creating a permissions policy$] given |navigable|'s [=fenced navigable container=], |origin|,
      and <var ignore>fenced</var> set to true.
 
-     Note: This is identical to the [=Document/permissions policy=] that will be created when the
-     navigation constructs the ultimate {{Document}} for this pending navigation. We create it now
-     and run tests on it since this is the appropriate time to determine if a navigation will fail,
-     and then throw it away. If the navigation succeeds, it will be recreated identically and
-     unconditionally installed on the {{Document}}.
+     Note: This is almost identical to the [=Document/permissions policy=] that will be created when
+     the navigation constructs the ultimate {{Document}} for this pending navigation (with the
+     [=Create a Permissions Policy for a fenced navigable=] algorithm). We create it now and run
+     tests on it since this is the appropriate time to determine if a navigation will fail, and then
+     throw it away. If the navigation succeeds, it will be recreated and unconditionally installed
```

Comment:
If the `allow` attribute contains `a, b, c, x, y, z`, but the config instance only contains `x, y, z`, then the permission policy created in _this_ algorithm will contain `a, b, c, x, y, z`, and we'll throw it away, right? Can we make it more clear that this permission policy has that characteristic? That's only implicitly alluded to via:

> However, the recreation will not include any additional enabled permissions that are not included in the [=fenced frame config/effective enabled permissions=],

But I think we can make it crystal clear and explicit by just saying early on in this paragraph that this, just like when it's called on iframes, will include all of the `allow` permissions, even those that are _not_ in the config. And then we can go into the second half of this paragraph where we talk about the fact that this is OK because once we actually create the real one, we just use the config's permissions, which does _not_ include the extra ones specified outside.